nginx/default

```diff
@@ -15,7 +15,7 @@ server {
 
     # Add index.php to the list if you are using PHP
     index index.html index.htm index.nginx-debian.html;
-    server_name gluegun.minio.io; # managed by Certbot
+    server_name gluegun.min.io; # managed by Certbot
```

Comment:
Similar to my previous comment, gluegun.minio.io leads to our Docs site, but gluegun.min.io doesn't lead anywhere. If this is the setting to configure this redirect, this comment can be ignored.

Reply:
gluegun.min.io is added to our DNS @kaankabalak